### DISCUSSION TITLE
Cannot Start App Due to Postgres Error

### DISCUSSION THREAD

**Discussion by xFGhoul** [fastapi/full-stack-fastapi-template#1401] [Questions]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched in the documentation/README.
- [X] I already searched in Google "How to do X" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs/README and didn't find an answer.

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```Text
None for this case.
```

### Description

I am trying to start this app using `docker compose -f docker-compose.yml up -d`, but i run into an error where `prestart-1` times out after `300` seconds (I know that's how its configued), and when viewing the logs of `prestart`, I get:
```
WARNING:__main__:Finished call to '__main__.init' after 89.046(s), this was the 89th time calling it.
prestart-1     | INFO:__main__:Starting call to '__main__.init', this is the 90th time calling it.
prestart-1     | ERROR:__main__:(psycopg.OperationalError) connection failed: connection to server at "172.18.0.2", port 5432 failed: FATAL:  password authentication failed for user "postgres"
```

now I thought i messed up the environment variables but everything is set and everything works, I even built the DB seperately to make sure `docker compose up --build --force-recreate --no-deps db` and it works, but when I try to run everything together back to the same problem, new to docker so I may be doing something wrong here...

### Operating System

Linux

### Operating System Details

Ubuntu 24.10

### Python Version

3.12.7

**Comment by chriskmps** [comment]
Confirm this is still an issue with latest version

**Reply by xFGhoul** [reply]
Hey, I won't be at my desk for a couple hours, and I am not even using fastapi anymore, but this issue relates to having the psql service running on windows at the same time as trying to run postgres in a docker container, I am not 100% sure of the specifics but it causes conflicts, disabling the service beforehand makes everything work as intended.

**Comment by hamadaxx** [comment]
Hey, did you check that the POSTGRES_PASSWORD in the .env file is the same as the password of your pastgres db ?
